src/javascript/app/base/header.js

```diff
@@ -51,7 +51,8 @@ const Header = (() => {
 
     const setHeaderUrls = () => {
         const btn__signup = getElementById('btn__signup');
-        const signup_url = `https://deriv.${getTopLevelDomain()}/signup/`;
+        const current_language = Language.get().toLowerCase();
+        const signup_url = `https://deriv.${getTopLevelDomain()}/${current_language}/signup/`;
```

Comment:
Couple of change requests:
- There are other occurrences of redirection to signup page that seems to be missing from here (purchase.js)
- No need to pass `en` as english is the default language
- Need to consider languages with `_` and replace them with `-` (i.e. zh_cn)

I suggest to add a global function same as `getStaticUrl` in Deriv.app repo and reuse it.